tests/scheduler/test_scheduler.py

```diff
@@ -20,9 +20,9 @@
                 backend=BackendEnum.VLLM,
                 backend_parameters=[],
             ),
-            ValueError,
-            "The model contains custom code that must be executed to load correctly. If you trust the source, please pass the backend parameter `--trust-remote-code` to allow custom code to be run.",
```

Comment:
Why this change?  Shouldn't we give this info to users so that they can add `--trust-remote-code` for this case?

Reply:
If we obtained the pretrained_config through the fallback method, then we don't need --trust-remote-code. User will get this info while the fallback still can't obtained the result.

Reply:
So `PretrainedConfig.from_dict(config_dict)` will fail if `--trust-remote-code` is not provided?